hub/models.py

```diff
@@ -2527,44 +2553,60 @@ def get_record_field(self, record, field: str, field_type=None):
 
         return value
 
+    def record_url_template(self) -> Optional[str]:
+        """
+        Get the URL template for a record in the remote system.
+        """
+        return "https://admin.mailchimp.com/audience/contact-profile?contact_id={record_id}"
+
+    def record_url(self, record_id: str) -> Optional[str]:
+        """
+        Get the URL of a record in the remote system.
+        """
+        # TODO: figure this out – the record_id does not work
+        return "https://admin.mailchimp.com/audience/"
```

Comment:
Docs talk about using `the MD5 hash of the lowercase version of the list member's email address` for contact IDs sometimes. We could try this instead of the record_id.

I will add some code for this.